swap/config.go

```diff
@@ -27,12 +27,9 @@ const (
 	// DefaultPaymentThreshold is set to be equivalent to requesting and serving 10mb of data (2441 chunks (4096 bytes) = 10 mb, 10^7 bytes = 10 mb)
 	DefaultPaymentThreshold    = 2441*RetrieveRequestPrice + (10^7)*ChunkDeliveryPrice // 4096 * 2441 = 10 mb,
 	DefaultDisconnectThreshold = 20 * DefaultPaymentThreshold
-	// DefaultInitialDepositAmount is the default amount to send to the contract when initially deploying
+	// DefaultDepositAmount is the default amount to send to the contract when initially deploying
```

Comment:
Sure. 
It was initialDeposit, because the flag *only* applied when you booted up Swarm for the very first time. 
I had to change this because due to the change in depositing, the deposit action is not any more atomic with  the deployment action. If we would make the deposit only possible during the first boot, the user would have a problem if for some reason the deposit would not go through while the deployment went through.